### PR TITLE
add support for include and exclude options for better control over the transformation

### LINKED ISSUE #11: Add support for .env.defaults
I have found having a .env.defaults file pretty useful: https://github.com/mrsteele/dotenv-defaults

### PR DESCRIPTION
**Pull request by bencao**
Adds control over which env variables to transform.
Is a potential solution to fix #11 

@dac09 please help review

@tysonmatanich please verify whether this can be a general solution for your requested support for .env.defaults.

### CODE REVIEW

File: index.js
Comment (dac09) [inline index.js:9]: I'm not sure how the plugin behaves, but if you supply an empty `include` array will it include nothing or will it just include everything?

Explicitly destructuring the options, and adding the relevant properties might be necessary!

Comment (bencao) [reply index.js:9]: According to my local tests:

- if `include` is an empty array it means none of variables would be transformed
- if `include` is null it will transform all variables

Comment (bencao) [reply index.js:9]: @dac09 if no more concerns can we continue to merge?

Comment (dac09) [reply index.js:9]: Sounds good :)

**Review by dac09 — APPROVED** [review]

File: index.js
Comment (dac09) [reply index.js:9]: @bencao approved :) Not sure if you wanted me to merge or not, I'll leave it to you since you probably want to publish it too!

Comment (bencao) [reply index.js:9]: thanks, I'll merge and publish a new version.

**Pull request merged**
